templates/index.md.tmpl

```diff
@@ -88,3 +88,6 @@ Finally, go back to your stack page, and go to "Details" for Loki to get
 ```http
 https://grafana.com/orgs/<org-slug>/hosted-logs/<logs-instance-id>
 ```
+
+### Installing a plugin
```

Comment:
For a resource like this, the resource page is sufficient. No need for it to be on the index. My own resources were like "glue" for different provider instances, so that's why I put them there